vue.config.js

```diff
@@ -1,16 +1,30 @@
 process.env.VUE_APP_VERSION = require("./package.json").version;
+const fs = require("fs");
 
 // do not add trailing forward slash
-process.env.VUE_APP_API_BASE_URL =
-    process.env.NODE_ENV === "production"
-        ? "https://uc4.cs.uni-paderborn.de/api/production"
-        : "https://uc4.cs.uni-paderborn.de/api/develop";
+process.env.VUE_APP_API_BASE_URL = process.env.NODE_ENV === "production" ? "https://uc4.cs.uni-paderborn.de/api/production" : "/api";
+
+const endpoint = "https://uc4.cs.uni-paderborn.de/api/experimental/";
+// const endpoint = "https://uc4.cs.uni-paderborn.de/api/development/"
```

Comment:
For the merge on develop, we should use the endpoint "develop". (As soon as the changes are present there)

Reply:
yes, see checklist in PR template